app/src/main/java/com/github/se/wanderpals/ui/screens/trip/Map.kt

```diff
@@ -145,35 +189,48 @@ fun Map(mapViewModel: MapViewModel, client: PlacesClient) {
           },
           placeholder = { Text("Search a location") },
           trailingIcon = {
-            // if the search text is empty, show the search icon, otherwise show the clear icon
+            // if the search text is empty, show the search icon, otherwise show the clear
+            // icon
             if (searchText.isEmpty()) {
               Icon(
                   imageVector = Icons.Default.Search,
                   contentDescription = Icons.Default.Search.name,
-                  modifier = Modifier.testTag("searchButtonIcon").size(24.dp))
+                  modifier = Modifier.size(24.dp).testTag("searchButtonIcon"))
             } else {
               IconButton(
                   modifier = Modifier.testTag("clearSearchButton"), onClick = { searchText = "" }) {
                     Icon(
                         imageVector = Icons.Default.Clear,
                         contentDescription = Icons.Default.Clear.name,
-                        modifier = Modifier.testTag("clearSearchButtonIcon").size(24.dp))
+                        modifier = Modifier.size(24.dp).testTag("clearSearchButtonIcon"))
                   }
             }
           },
           leadingIcon = {
             Icon(
                 imageVector = Icons.Default.Menu,
                 contentDescription = Icons.Default.Menu.name,
-                modifier = Modifier.testTag("menuButtonIcon").size(24.dp))
+                modifier = Modifier.size(24.dp).testTag("menuButtonIcon"))
           }) {
-            // if the search bar is expanded when clicked modify the search text, show the list of
-            // suggestions
+            // if the search bar is expanded when clicked modify the search text, show the
+            // list of suggestions
             if (expanded) {
               listOfPropositions.forEach {
                 val primaryText = it.getPrimaryText(null).toString()
                 val placeId = it.placeId
-                val placeFields = listOf(Place.Field.LAT_LNG)
+
+                val placeFields =
```

Comment:
As discussed during the research that we have done together in person, the tags are interesting and relevant.